sdk/ruby/lib/chain/batch_response.rb

```diff
@@ -1,9 +1,3 @@
-def ensure_key_sorting(h)
```

Comment:
NBD, but would help to put this type of thing in a separate PR. I know it's a bit more work, but in the long-run we'll get a tighter commit history.